src/features/game/lib/processEvent.ts

```diff
@@ -8,7 +8,7 @@ const maxItems: Inventory = {
   // Seed limits + buffer of 20
   Sunflower: new Decimal("3000"),
   Potato: new Decimal("2000"),
-  Pumpkin: new Decimal("1000"),
+  Pumpkin: new Decimal("1500"),
```

Comment:
this need a contract upgrade as the current session contract limits to 1000, seeds are fine tho, the contract accepts 500 
pumpkin seeds.

Reply:
@mGranTY That's why I said it needs a blochain (should be contract) update in the main post